app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -0,0 +1,68 @@
+package org.fossasia.openevent.app.event.attendees;
+
+import android.content.Context;
+
+import org.fossasia.openevent.app.data.EventDataRepository;
+import org.fossasia.openevent.app.data.UtilModel;
+import org.fossasia.openevent.app.data.contract.IEventDataRepository;
+import org.fossasia.openevent.app.data.contract.IUtilModel;
+import org.fossasia.openevent.app.data.models.Attendee;
+import org.fossasia.openevent.app.event.attendees.contract.IAttendeesPresenter;
+import org.fossasia.openevent.app.event.attendees.contract.IAttendeesView;
+
+public class AttendeesPresenter implements IAttendeesPresenter {
+
+    IEventDataRepository eventDataRepository;
+    IAttendeesView attendeesView;
+    Long eventId;
+
+    public AttendeesPresenter(IAttendeesView attendeesView, Context context, Long eventId) {
+        this.attendeesView = attendeesView;
+        eventDataRepository = new EventDataRepository(new UtilModel(context));
+        this.eventId = eventId;
+    }
+
+    @Override
+    public void attach() {
+        loadAttendees(true);
```

Comment:
Just imagine how much user will have to wait for every attendee information? This is an anti-pattern to always fetch from the internet. Suppose, there are 10,000 attendees. How much time it will take to download that info if we don't use the cached versions